inference-engine/tests/functional/plugin/cpu/shared_tests_instances/single_layer_tests/gather.cpp

```diff
@@ -15,26 +15,81 @@ const std::vector<InferenceEngine::Precision> netPrecisions = {
         InferenceEngine::Precision::FP32,
```

Comment:
Please add other precisions like I8, FP16, BF16, I64.

Reply:
Added, thanks!

I didn't add I8 because Gather [aren't supported](https://github.com/openvinotoolkit/openvino/blob/61108f1147ea9a022b25c1eaf1358268e97ecfad/ngraph/core/src/op/util/gather_base.cpp#L235) it in ngraph reference

If I'm wrong, correct me please

Reply:
Added, thanks!

I didn't add I8 because Gather [aren't supported](https://github.com/openvinotoolkit/openvino/blob/61108f1147ea9a022b25c1eaf1358268e97ecfad/ngraph/core/src/op/util/gather_base.cpp#L235) it in ngraph reference (he wrote "`Interpreter backend doesn't implement evaluate method for OP Gather`"

If I'm wrong, correct me please

Reply:
I assume right way is to add INT8 here, add appropriate row into skip config and create ticket to support it in reference.

Reply:
Yes, you're right. Added this in commit: 3f8e333baa3adba732925ce2399a2b1992eb9247
Thank you!